insta_user_names-filter/insta_user_names_filter.go

```diff
@@ -0,0 +1,41 @@
+package filter
+
+import (
+	"encoding/json"
+
+	"github.com/codeuniversity/smag-mvp/kafka/changestream"
+	"github.com/segmentio/kafka-go"
+)
+
+type user struct {
+	ID       int    `json:"id"`
+	UserName string `json:"user_name"`
+}
+
+// UserNamesFilter represents the UserNamesFilter containing all clients it uses
+type UserNamesFilter struct {
+	*changestream.Filter
+}
+
+// New returns an initilized UserNamesFilter
+func New(kafkaAddress, kafkaGroupID, changesTopic, userNameTopic string) *UserNamesFilter {
```

Comment:
Why wrap the Filter into a new type? Can't we directly return the `changestream.NewFilter(...)`?